src/Workspaces/Core/Desktop/Workspace/SQLite/SQLite.cs

```diff
@@ -0,0 +1,4003 @@
+﻿//
```

Comment:
:bulb: Since this file is externally-sourced, consider adding the following line to the top:

```csharp
// <auto-generated>
//   Externally sourced file. Mark as generated to disable code style analysis.
// </auto-generated>
```